tract_querier/tract_math/decorator.py

```diff
@@ -196,7 +196,7 @@ def process_output(output, file_output=None):
             header = dict(zip(writer.fieldnames, writer.fieldnames))
             writer.writerow(header)
 
-        first_value = output.values()[0]
+        first_value = list(output.values())[0]
```

Comment:
`next(iter(output.values()))` might be more efficient here

Reply:
Changed in 24d8aaf.